client/src/modules/Opportunities/components/EditCv/form-validation.ts

```diff
@@ -78,6 +78,17 @@ export const contactsValidationRules: {
       max: 30,
       message: validationMessages.max(30),
     },
+    () => ({
+      async validator(_, value) {
+        /* https://github.com/GrantBirki/github-username-regex-js */
+        const githubUsernamePattern =
+          /^(?:[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}|[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*(_[a-zA-Z0-9]+))$/i;
```

Comment:
please move to `client/src/services/validators.ts` file

Reply:
also it's wrong to use this regex as it checks for github username and [GitHub EMU](https://docs.github.com/en/enterprise-cloud@latest/admin/identity-and-access-management/understanding-iam-for-enterprises/about-enterprise-managed-users) username.

It's better to go with something simpler like that:
```
^[a-zA-Z0-9](?:[a-zA-Z0-9]|-(?=[a-zA-Z0-9])){0,37}[a-zA-Z0-9]$
```